packages/components/src/components/Dropdown/index.tsx

```diff
@@ -65,7 +65,7 @@ const Menu = styled.ul<MenuProps>`
     min-width: ${props => props.minWidth}px;
     box-shadow: 0 2px 7px 0 ${props => props.theme.BOX_SHADOW_BLACK_15},
         0 2px 3px 0 ${props => props.theme.BOX_SHADOW_BLACK_5};
-    z-index: 2;
+    z-index: ${variables.Z_INDEX.GUIDE_PANEL + 2};
```

Comment:
?

Reply:
dropdown z-index should be greater than `guidePanel`

Reply:
if we started using those relative z-indexes, we would loose the main benefit of having z-indexes defined on one place - if you would like to add some new element above guide panel, you can't see clearly that there is some Dropdown...

What about to define Z_INDEX variable for that (e.g. Z_INDEX.DROPDOWN or Z_INDEX.DROPDOWN_MENU?) and you could possibly add there a comment, that it should lay above guide panel...

Reply:
I added `DROPDOWN_MENU` variable in a7b4d1160fcc075eda5a229a4e910bdbeb6e818a